src/components/PasswordInput/PasswordInput.js

```diff
@@ -0,0 +1,47 @@
+import { InputAdornment } from "@material-ui/core";
+import { withStyles } from "@material-ui/core/styles";
+
+import TextField from "@material-ui/core/TextField";
+import { RemoveRedEye } from "@material-ui/icons";
+import PropTypes from "prop-types";
+import React, { useState } from "react";
+
+const styles = theme => ({
+  eye: {
+    cursor: "pointer"
+  }
+});
+
+const PasswordInput = props => {
+  const [passwordIsMasked, setPasswordIsMasked] = useState(true);
```

Comment:
just a matter of preference.. I usually start my boolean variables with `is` e.g ìsPasswordMasked` in this case..

Reply:
me too...well spotted. I was following a tutorial about material ui and I just missed that. I'll change it